lib/streamlit/ReportSession.py

```diff
@@ -107,6 +109,8 @@ def __init__(self, ioloop, script_path, command_line):
 
         self._scriptrunner = None
 
+        self.fileManager = FileManager()
```

Comment:
should be `self._file_manager` (snake-case, and also a leading underscore since it's a private variable)